tasklist/client/src/v1/TaskDetails/index.test.tsx

```diff
@@ -396,14 +396,14 @@ describe('<Task />', () => {
   });
 
   it('should complete task without variables', async () => {
+    let isCompleted = false;
+
     nodeMockServer.use(
-      http.get(
-        '/v1/tasks/:taskId',
-        () => {
-          return HttpResponse.json(taskMocks.assignedTask());
-        },
-        {once: true},
-      ),
+      http.get('/v1/tasks/:taskId', () => {
+        return HttpResponse.json(
+          isCompleted ? taskMocks.completedTask() : taskMocks.assignedTask(),
+        );
+      }),
```

Comment:
This didn't address the comment I did before

The whole point of using MSW is to have a declarative API do mock HTTP requests

In this case you need to call `http.get('/v1/tasks/:taskId',` in the beginning with the assigned task response

and then you call it again after clicking on the complete button but this time returning the completed task

For the first one you can mock it only once but for the second one you might need to not use the mock only once because of the polling